modules/apps/object/object-admin-rest-impl/src/main/java/com/liferay/object/admin/rest/internal/jaxrs/exception/mapper/RequiredObjectFieldExceptionMapper.java

```diff
@@ -0,0 +1,62 @@
+/**
+ * Copyright (c) 2000-present Liferay, Inc. All rights reserved.
+ *
+ * This library is free software; you can redistribute it and/or modify it under
+ * the terms of the GNU Lesser General Public License as published by the Free
+ * Software Foundation; either version 2.1 of the License, or (at your option)
+ * any later version.
+ *
+ * This library is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE. See the GNU Lesser General Public License for more
+ * details.
+ */
+
+package com.liferay.object.admin.rest.internal.jaxrs.exception.mapper;
+
+import com.liferay.object.exception.RequiredObjectFieldException;
+import com.liferay.portal.kernel.language.Language;
+import com.liferay.portal.kernel.util.LocaleUtil;
+import com.liferay.portal.vulcan.jaxrs.exception.mapper.BaseExceptionMapper;
+import com.liferay.portal.vulcan.jaxrs.exception.mapper.Problem;
+
+import javax.ws.rs.core.Response;
+import javax.ws.rs.ext.ExceptionMapper;
+
+import org.osgi.service.component.annotations.Component;
+import org.osgi.service.component.annotations.Reference;
+
+/**
+ * @author Daniel Bonasser
+ */
+@Component(
+	property = {
+		"osgi.jaxrs.application.select=(osgi.jaxrs.name=Liferay.Object.Admin.REST)",
+		"osgi.jaxrs.extension=true",
+		"osgi.jaxrs.name=Liferay.Object.Admin.REST.RequiredObjectFieldExceptionMapper"
+	},
+	service = ExceptionMapper.class
+)
+public class RequiredObjectFieldExceptionMapper
+	extends BaseExceptionMapper<RequiredObjectFieldException> {
+
+	@Override
+	protected Problem getProblem(
+		RequiredObjectFieldException requiredObjectFieldException) {
+
+		return new Problem(
+			Response.Status.BAD_REQUEST,
+			_language.get(
+				LocaleUtil.getDefault(),
```

Comment:
In the others exceptionMapper of `object-admin-rest-impl`, The ones that use localized message have a slightly different pattern.

1. Create the variable
```
 	@Context
	private AcceptLanguage _acceptLanguage;
```
Then use `_acceptLanguage.getPreferredLocale()`, instead `LocaleUtil.getDefault()`


2. The hard-coded message key use to be defined inside Exception class (**-api** module), and call here through `getMessageKey()`